testutil/compose/static/lodestar/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+while ! curl "http://${NODE}:3600/up" 2>/dev/null; do
+  echo "Waiting for http://${NODE}:3600/up to become available..."
+  sleep 5
+done
+
+echo "Creating testnet config"
+rm -rf /tmp/testnet || true
+mkdir /tmp/testnet/
+curl "http://${NODE}:3600/eth/v1/config/spec" | jq -r .data | yq -P > /tmp/testnet/config.yaml
+
+for f in /compose/"${NODE}"/validator_keys/keystore-*.json; do
+    echo "Importing key ${f}"
+
+    # Import keystore with password.
+    node /usr/app/packages/cli/bin/lodestar validator import \
+        --network="dev" \
+        --presetFile="/tmp/testnet/config.yaml" \
```

Comment:
Only the network is relevant here as it is used to determine to which folder to store the keystores, `--presetFile` and `--paramsFile` can be removed.